packages/default/scss/dialog/_layout.scss

```diff
@@ -54,51 +54,18 @@
         border-width: 1px 0 0;
         border-style: solid;
         border-color: inherit;
-        background-color: transparent;
         display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+        align-items: center;
         flex: 0 0 auto;
 
         .k-button {
-            // sass-lint:disable no-important
+            width: auto !important; // sass-lint:disable-line no-important
```

Comment:
Please document uses of `!important` with an inline comment. There should be a reason why are we prohibiting customers from setting the button width. While this rule was not added in this PR, it is a good time to explain why we need this. I personally don't recall, and future maintainers won't know, too.

Reply:
I have added an explaining comment. The reason is to prevent buttons to stretch when "stretched button layout" is set as Material guidelines do not support such appearance.